cmd/soroban-cli/Cargo.toml

```
@@ -26,6 +26,10 @@ name = "soroban_cli"
 path = "src/lib.rs"
 doctest = false
 
+[features]
+default = []
+opt = ["wasm-opt"]
```

This change as it is, is actually adding two new features to the CLI: `opt` and `wasm-opt`. We can make it one new feature by referencing the `wasm-opt` create as a `dep:` feature, which unintuitively tells cargo we don't want the implicitly `wasm-opt` feature to exist.
```suggestion
opt = ["dep:wasm-opt"]
```